src/irmin-pack/irmin_pack_layers.ml

```diff
@@ -726,42 +728,9 @@ struct
           nodes (or contents) not yet attached to a commit (or a node resp.), so
           we have to iter over the graph of commits, iter over the graph of
           nodes, and lastly iter over contents. *)
```

Comment:
can you rewrite the comments above to reflect this changes? we no longer need to iter over nodes and contents separately, as it is suggested.